examples/inputs/example_01a/exchanges_units.csv

```diff
@@ -0,0 +1,3 @@
+name,technology,direction,bidding_EOM,max_power,min_power,price,unit_operator
+import,n/a,import,naive_eom,1000000,0,0,eom_de
+export,n/a,export,naive_eom,1000000,0,2999,eom_de
```

Comment:
if min_power == max_power - doesn't that mean, that we always require  to export `1000000` for a price of 2999? This does not look that reasonable?

But I did not run the simulation yet.